web/src/components/molecules/Content/ContentForm/index.tsx

```diff
@@ -0,0 +1,181 @@
+import styled from "@emotion/styled";
+import { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import InputNumber from "@reearth-cms/components/atoms/InputNumber";
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
+import Select from "@reearth-cms/components/atoms/Select";
+import TextArea from "@reearth-cms/components/atoms/TextArea";
+import Upload from "@reearth-cms/components/atoms/Upload";
+import { ItemField } from "@reearth-cms/components/molecules/Content/types";
+import { FieldType, Model } from "@reearth-cms/components/molecules/Schema/types";
+import { useT } from "@reearth-cms/i18n";
+import { validateURL } from "@reearth-cms/utils/regex";
+
+export interface Props {
+  itemId?: string;
+  initialFormValues: any;
+  defaultFormValues: any;
+  model?: Model;
+  onItemCreate: (data: { schemaID: string; fields: ItemField[] }) => Promise<void>;
+  onItemUpdate: (data: { itemID: string; fields: ItemField[] }) => Promise<void>;
+  onBack: () => void;
+}
+
+const ContentForm: React.FC<Props> = ({
+  itemId: itemID,
```

Comment:
Why do you need to change the name? If possible, please keep it `itemId` and keep it consistent throughout the code.

Reply:
@KaWaite , so it will be easier for me to remember to refactor and to refactor when the backend is done fixing the variable name.
I will rename the variable when @mimoham24 PR is approved.

Reply:
Ah the backend is changing the name as well? Ok